packages/mds-ingest-service/repository/migrations/1646013688688-DropMigrationColumns.ts

```diff
@@ -0,0 +1,51 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class DropMigrationColumns1646013688688 implements MigrationInterface {
+  name = 'DropMigrationColumns1646013688688'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Do we really want to drop all these columns? I feel like it could be useful for posterity -- in addition to the case that we have to support a similar style migration again in the future.

Reply:
Would it be a big deal to add them again? Also what if none of us are around when the next migration needs to happen and nobody remembers what these columns are for?

Reply:
I agree with @avatarneil … I think it may prove useful to retain these columns although it probably is fine to remove the indexes.